cmd/pmgr/main.go

```diff
@@ -1,11 +1,178 @@
 package main
 
 import (
+	"bytes"
+	"flag"
 	"fmt"
-
-	"github.com/hftamer/go-training/pkg/pmgr"
+	"golang.org/x/crypto/bcrypt"
+	"log"
+	"os"
+	"strings"
 )
 
+type userMap map[string]string
+
 func main() {
-	fmt.Println(pmgr.GetQuote())
+	userData := userMap{}
+	filename := "userData.txt"
+	populateUserMapWithDataFromFile(filename, userData)
+	runCommandLineProgram(userData, filename)
+}
+
+func runCommandLineProgram(userData userMap, fileName string) {
+	addHelperFlagText()
```

Comment:
In general, functions should follow [SRP](https://en.wikipedia.org/wiki/Single-responsibility_principle). Create a `ParseArgs` function that parses the argument for you and avoid calling it in this function.